frontend/src/views/main/project/TestSuiteExecutions.vue

```diff
@@ -154,6 +154,11 @@ const executionBreadcrumbs = computed(() =>
       }
```

Comment:
It's not part of the current PR scope, but `date` filter could be reused here too